### PR TITLE
Issue#15607-Audio/Ambience

### PR DESCRIPTION
**Pull request by HighTechPuddle**
## About the PR
I have converted all license.txt to attributions.yml within Audio/Ambience and its subfolders. Everything is verified but I would still appreciate if someone went over it. **I left the license.txt files in there just for ease of access when verifying please notify me when its ready to merge and I will delete them.**

Of course the converting wasn't problem less. Here they are:

- shipambience.ogg - Couldn't verify license: CC-BY-SA
- /Antag - traitor_start.ogg - Wasn't in attributions.yml
- /Objects - buzzing.ogg, emf_buzz.ogg, engine_hum.ogg, gravity_gen_hum.ogg, hdd_buzz.ogg, light_hum.ogg, periodic_beep.ogg, vending_machine_hum.ogg - Wasn't in attributions.yml or license.txt

(I am sorry about last time I did a really bad job and neglected a lot of things just to make it easier for myself. This time I will do it as I was told, like splitting it into multiple PRs/branches. It will take longer tho.)

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

File: Resources/Audio/Ambience/attributions.yml [hunk 1]
```diff
@@ -3,63 +3,27 @@
...
-  - "ambiruin7.ogg"
-  - "ambisin1.ogg"
-  - "ambisin2.ogg"
-  - "ambisin3.ogg"
-  - "ambisin4.ogg"
-  - "ambitech.ogg"
-  - "ambitech2.ogg"
-  - "ambitech3.ogg"
-  - "maintambience.ogg"
+- files: ["ambiatmos.ogg, ambiatmos2.ogg, ambicave.ogg, ambicha1.ogg, ambicha2.ogg, ambicha3.ogg, ambicha4.ogg, ambidanger.ogg, ambidanger2.ogg, ambigen1.ogg, ambigen2.ogg, ambigen3.ogg, ambigen4.ogg, ambigen5.ogg, ambigen6.ogg, ambigen7.ogg, ambigen8.ogg, ambigen9.ogg, ambigen10.ogg, ambigen11.ogg, ambigen12.ogg, ambigen13.ogg, ambigen14.ogg, ambigen15.ogg, ambiholy.ogg, ambiholy2.ogg, ambiholy3.ogg, ambilava1.ogg, ambilava2.ogg, ambilava3.ogg, ambimaint1.ogg, ambimaint2.ogg, ambimaint3.ogg, ambimaint4.ogg, ambimaint5.ogg, ambimine.ogg, ambimo1.ogg, ambimo2.ogg, ambimystery.ogg, ambinice.ogg, ambiodd.ogg, ambireebe1.ogg, ambireebe3.ogg, ambiruin.ogg, ambiruin2.ogg, ambiruin3.ogg, ambiruin4.ogg, ambiruin5.ogg, ambiruin6.ogg, ambiruin7.ogg, ambisin1.ogg, ambisin2.ogg, ambisin3.ogg, ambisin4.ogg, ambitech.ogg, ambitech2.ogg, ambitech3.ogg, maintambience.ogg"]
```
Comment (metalgearsloth) [inline Resources/Audio/Ambience/attributions.yml:6]: Please leave these as is.

File: Resources/Audio/Ambience/Antag/attributions.yml [hunk 2]
```diff
@@ -1,12 +1,12 @@
-- files: ["﻿nukeops_start.ogg"]
+- files: ["nukeops_start.ogg"]
   license: "CC-BY-NC-SA-3.0"
-  copyright: "Taken from TG station."
-  source: "https://github.com/tgstation/tgstation/commit/827967c9650c23af64280ad1491405fed8f644c5"
+  copyright: "Taken from TG station"
+  source: "https://github.com/tgstation/tgstation/commit/827967c9650c23af64280ad1491405fed8f644c5#diff-6cc910b7cad9ac4333c8d0885fc844746066120b465d8d4ba8f7019169316574"
```
Comment (metalgearsloth) [inline Resources/Audio/Ambience/Antag/attributions.yml:4]: Sound isn't TG the PR from last week made a mistake.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Resources/Audio/Ambience/Antag/attributions.yml (hunk 2 above)
Comment (HighTechPuddle) [reply Resources/Audio/Ambience/Antag/attributions.yml:4]: What you mean? The sounds are the same and it isnt mentioned in their attributions or license files. So it has to be from tgstation.

**Comment by HighTechPuddle** [comment]
Just updated it because build failed.

**Comment by HighTechPuddle** [comment]
Does someone know why it keeps failing?

**Comment by metalgearsloth** [comment]
merge master

File: Resources/Audio/Ambience/Antag/attributions.yml (hunk 2 above)
Comment (metalgearsloth) [reply Resources/Audio/Ambience/Antag/attributions.yml:4]: It's not NC

Comment (HighTechPuddle) [reply Resources/Audio/Ambience/Antag/attributions.yml:4]: Oh you mean the license is wrong? Alr I will fix when I get back from school.

**Review by metalgearsloth — APPROVED** [review]

**Comment by HighTechPuddle** [comment]
oi the issues still need addressing

**Pull request merged**